docs/metrics-details.md

```diff
@@ -0,0 +1,22 @@
+---
+id: metrics-details
+title: Metrics Details
+sidebar_label: Metrics Details
+---
+
+## Introduction
+
+RIG is providing it's Metrics as [**"Prometheus"**](https://prometheus.io) format. Prometheus is CNCFs second graduated project and therefore can be assumed as de-facto standard.
+
+This metrics can be used to be included into a monitoring platform like [**Grafana**](https://grafana.com/)
+
+## RIG Specific Metrics
+
+Currently there are no RIG specific metrics implemented yet. We are exposing the standard metrics providing information about the running application itself.
```

Comment:
I think it might be useful to list here what we support (at least core metrics) - detailed info can be found in mentioned link

Reply:
Also once we have more specific metrics we should create some example setup since it's not that straightforward process.